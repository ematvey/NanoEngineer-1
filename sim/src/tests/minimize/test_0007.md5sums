stdout F45D33A7F1CFE3CC33B5FC260FD9F9DA
stderr D41D8CD98F00B204E9800998ECF8427E
