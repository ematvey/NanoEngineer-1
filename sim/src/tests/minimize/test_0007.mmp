mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (0.988510, 0.095252, 0.028494, -0.113860) (10.000000) (-3.966752, -2.232186, 3.919845) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (Untitled)
mol (Chunk-C1) def
atom 1 (6) (4324, 860, 1629) def
atom 2 (0) (4557, 347, 2153) def
bond1 1
atom 3 (0) (4119, 498, 981) def
bond1 1
atom 4 (6) (3067, 1636, 2248) def
bond1 1
atom 5 (0) (2592, 1146, 2605) def
bond1 4
atom 6 (0) (2697, 1994, 1675) def
bond1 4
atom 7 (0) (3343, 2145, 2756) def
bond1 4
mol (Chunk-C6) def
atom 8 (0) (4966, 9816, 3035) def
atom 9 (0) (5557, 8867, 2463) def
atom 10 (6) (5222, 9094, 3118) def
bond1 8 9 1
atom 11 (0) (5667, 9012, 3741) def
bond1 10
egroup (Untitled)
end1
end molecular machine part Untitled
