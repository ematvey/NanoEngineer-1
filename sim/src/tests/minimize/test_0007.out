======= test_0007.test =======
TYPE fail
PROGRAM simulator -f300 -m -X -x test_0007.mmp

